src/lib/useImpliedTicket.js

```diff
@@ -0,0 +1,14 @@
+import * as ob from 'urbit-ob';
+
+import { prependSig } from './transformers';
+
+/**
+ * pull the suggested ticket from the #hash in the url
+ */
+export default function useImpliedTicket() {
+  const hash = window.location.hash.slice(1);
+  const tick = prependSig(hash);
```

Comment:
`ticket`, to be clear?

Reply:
You are clearly not a hoon programmer. (^:

Reply:
😂